tests/conftest.py

```diff
@@ -70,12 +71,17 @@ def run_migrations() -> Iterator[None]:
         if isinstance(schema, WritableTableSchema):
             table_name = schema.get_local_table_name()
 
-            nodes = [*cluster.get_local_nodes(), *cluster.get_distributed_nodes()]
-            for node in nodes:
-                connection = cluster.get_node_connection(
-                    ClickhouseClientSettings.MIGRATE, node
-                )
-                connection.execute(f"TRUNCATE TABLE IF EXISTS {database}.{table_name}")
+            try:
+                nodes = [*cluster.get_local_nodes(), *cluster.get_distributed_nodes()]
+                for node in nodes:
+                    connection = cluster.get_node_connection(
+                        ClickhouseClientSettings.MIGRATE, node
+                    )
+                    connection.execute(
+                        f"TRUNCATE TABLE IF EXISTS {database}.{table_name}"
+                    )
+            except Exception as e:
+                logging.info(f"could not tear down all nodes for {storage_key} {e}")
```

Comment:
What exception is actually expected here now? Is it just the connection error? If so I think we should just catch that specific one not any type of exception which could hide other issues.

Also, the try/except is around all of the nodes of the cluster not each individual one. Do you want to just give up on them all if 1 fails?